x-pack/metricbeat/module/aws/elb/elb.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package elb
+
+import (
+	"os"
+
+	"github.com/elastic/beats/metricbeat/mb"
+	xpackmb "github.com/elastic/beats/x-pack/metricbeat/mb"
+
+	// Register input module and metricset
+	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws"
+	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws/cloudwatch"
+)
+
+func init() {
+	// To be moved to some kind of helper
+	os.Setenv("BEAT_STRICT_PERMS", "false")
+	mb.Registry.SetSecondarySource(xpackmb.NewLightModulesSource("../../../module"))
+}
```

Comment:
This should be in a test file (move to `elb_test.go` for example), if not we are overriding strict perms and secondary source in the final binary.